isolate.sh

```diff
@@ -441,6 +405,28 @@ function removeNSS(){
         info "Namespace Scope CR \"odlm-scope-managedby-odlm\" not present. Moving on..."
     fi
 
+    failcheck=$(${OC} get nss --all-namespaces | grep nss-odlm-scope || echo "failed")
```

Comment:
We only need to check the original-cs-ns for NSS CRs to delete, not all namespaces, right?

Reply:
yeah, this code was grabbed from the conversion script so I'll need to change it there too